profiler/src/ProfilerEngine/Datadog.Profiler.Native.Linux/CrashReportingLinux.cpp

```diff
@@ -236,6 +237,20 @@ std::vector<StackFrame> CrashReportingLinux::GetThreadFrames(int32_t tid, Resolv
         // TODO: Check if the stacktrace is from the tracer or the profiler
         stackFrame.isSuspicious = false;
 
+        std::filesystem::path modulePath(module.first);
+
+        if (modulePath.has_filename())
+        {
+            const auto moduleFilename = modulePath.filename().string();
```

Comment:
Nit:
you can simplify it a bit and avoid one allocation by going for:
```
 const auto moduleFilename = modulePath.stem().string();

 // use rfind instead of find because it will search only once starting from the offset 0
 if (moduleFilename.rfind("Datadog", 0) == 0
     || moduleFilename == "libdatadog"
     || moduleFilename == "datadog"
     || moduleFilename == "libddwaf"
     || moduleFilename == "ddwaf"
     )
```